src/hooks/data/vaults/fallbacks/apr/useCurveVaultArbitrumTotalApr.ts

```diff
@@ -25,22 +35,30 @@ export default function useCurveVaultArbitrumTotalApr({
   return curveApiQuery
 }
 
+const respSchema = z.object({
```

Comment:
When using third-party APIs, it's a pretty good idea to use `zod` to validate the response. 

These APIs could change at any moment without notice and `zod` gives us a typesafe response to use.